imp/NonTerminals.cs

```diff
@@ -57,10 +57,304 @@ public class NonTerminals : INonTerminals {
                              lastNoVariantRecDependent = Production.TypeDeclarationTail,
                              firstOptionDependent = firstConstParamDependent,
                              lastOptionDependent = lastNoVariantRecDependent;
-    public static uint alternateSetOffset = (uint)lastOptionDependent - (uint)firstOptionDependent + 1;               
-    
-    TokenSet[] followSet = new TokenSet[Count()];
-    TokenSet[] firstSet = new TokenSet[Count()];
+    public static uint alternateSetOffset = (uint)lastOptionDependent - (uint)firstOptionDependent + 1;
+

```

Comment:
Please set your editor to expand tab into two spaces [ASCII(32)], do not let it insert any TABs [ASCII(9)].

Lines should be broken as follows

```
public Foobar foobar =
  foo = Foobar.Foo,
  bar = Foobar.Bar,
  baz = Foobar.Baz;
```
etc